pkg/util/metric/prometheus_exporter_test.go

```diff
@@ -61,23 +61,23 @@ func TestPrometheusExporter(t *testing.T) {
 	}
 
 	expected := map[string]family{
-		"one_gauge": {[]metricLabels{
+		"one.gauge": {[]metricLabels{
```

Comment:
could we add a test case instead that uses the new format instead of modifying the existing tests?

Reply:
Added a new test case below, using content negotiation to control the escaping behavior.